docker-compose.prod.yml

```diff
@@ -21,9 +23,11 @@ services:
       "--master",
       "--callable", "app",
       "--processes", "2",
+      "--logto", "/var/log/uwsgi.log",
```

Comment:
nit: Is there a reason we call it `uwsgi.log`? If not, maybe `api.log` is easier to understand?